synthesizer/process/src/stack/authorization/mod.rs

```diff
@@ -147,6 +148,73 @@ impl<N: Network> Authorization<N> {
             _ => false,
         }
     }
+
+    /// Checks whether the authorization is for a valid program edition.
+    pub fn check_valid_edition(&self, process: &crate::Process<N>, _consensus_version: ConsensusVersion) -> Result<()> {
```

Comment:
Nit: Why the underscore here? We usually use this naming pattern for unused variables

```suggestion
    pub fn check_valid_edition(&self, process: &crate::Process<N>, consensus_version: ConsensusVersion) -> Result<()> {
```

Reply:
Clippy doesn't recognize the variables are used past `#[cfg(not(any(test, feature = "test")))]`. 

A resolution would be to use the variables also under the opposite featureset. But that doesn't seem worth the trouble. It is awkward but harmless, and happens in a few places in the codebase.